hack/tools/install_kustomize.sh

```diff
@@ -2,7 +2,7 @@
 
 [[ -f bin/kustomize ]] && exit 0
 
-version=3.5.4
+version=4.4.1
```

Comment:
I will get rid of this file in a followup PR to ensure we dont use two different files for installing kustomize. Since this file is used in CI dockerfiles as well,I am keeping it as it is for now.